src/Application/Game.cpp

```diff
@@ -1706,14 +1751,20 @@ void Game::EventLoop() {
                     continue;
                 case UIMSG_RestWindow:
                     pMessageQueue_50CBD0->Flush();
+                    // toggle
```

Comment:
No need to commit this commented out code?

Reply:
Maybe it was left behind in the hopes that 8hour rest and rest menu hotkey conflict will be eventually resolved by making all hardcoded hotkeys configurable? So we can toggle rest dialog as well at the end?

Reply:
Yes, I would prefer to make rest toggleable eventually.